src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -413,9 +414,17 @@ const DataAccessRequestApplication = (props) => {
       }, Navigation.console(Storage.getCurrentUser(), props.history).response);
     } catch (_error) {
       setShowDialogSubmit(false);
-      NotyUtil.showError({
-        text: 'Data Access Request submission failed. Please save and try submitting again.'
-      });
+      Notifications.showError(
+          {
+            text: <ReactMarkdown>{_error.response.data.message}</ReactMarkdown>,
```

Comment:
We can probably rename `_error` as `error` since we are using it in the markdown now.